packages/jupyter-ai/style/rendermime-markdown.css

```diff
@@ -0,0 +1,6 @@
+.custom-markdown pre,
+.custom-markdown pre > code {
+  background-color: var(--jp-cell-editor-background);
+  overflow-x: auto;
```

Comment:
Does adding this on both elements introduce nested scrollbars? Will help to verify in different browsers.

Reply:
Reworked the css file to have a more straightforward structure.

Reply:
Answering the question, previously when adding this on both elements this was not introducing nested scrollbars based on my testing.